manager/Pipfile

```diff
@@ -6,6 +6,8 @@ verify_ssl = true
 [dev-packages]
 autopep8 = "~=1.5.4"
 black = "==19.10b0"
+coverage = "~=5.3"
+coveralls = "~=2.1.2"
```

Comment:
Is this still necessary?

Reply:
`coveralls` isn't. (I thought I removed that...)

`coverage`, though, is, because that is used to generate the coverage report, which is later uploaded to Coveralls.